### DISCUSSION TITLE
Problem connecting

### DISCUSSION THREAD

**Discussion by R8bert** [blackcowmoo/Grafana-Google-Analytics-DataSource#95] [Q&A]
Hi.
I've done every step and i get `CheckHealth: Not Exist Valid Profile` when i try to test.
Is there a problem ? and how i can solve it ?
Thanks!

**Comment by lcc3108** [comment]
Hello.
My reply is late.
1. Which data would you like to see in Google Analytics, UA or GA accounts?(UA will no longer receive data from 2023/07). )
2. In the grafana plug-in setting page, please make sure that you selected the correct UA or GA
